src/utils/media-utils.js

```diff
@@ -24,6 +24,23 @@ function b64EncodeUnicode(str) {
   return btoa(encodeURIComponent(str).replace(CHAR_RE, (_, p1) => String.fromCharCode("0x" + p1)));
 }
 
+const farsparkEncodeUrl = url => {
+  // farspark doesn't know how to read '=' base64 padding characters
+  // translate base64 + to - and / to _ for URL safety
+  return b64EncodeUnicode(url)
+    .replace(/=+$/g, "")
+    .replace(/\+/g, "-")
+    .replace(/\//g, "_");
+};
+
+export const scaledThumbnailUrlFor = (url, width, height) => {
+  if (process.env.FARSPARK_SERVER && process.env.NODE_ENV === "production") {
```

Comment:
oh, if there is a way to know if you are using a local reticulum node or not we could probably make it work. i was testing with the local reticulum instance, in which case you can't use dev farspark because the images are not internet accessible

Reply:
Oh right. Fair enough, we can change this later if we end up using local spoke + dev reticulum more often.

Reply:
I'm just going to add an explicit check for hubs.local